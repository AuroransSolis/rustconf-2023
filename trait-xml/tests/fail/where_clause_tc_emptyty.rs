trait_xml::trait_xml! {
    <trait>
        <name>Foo</name>
        <bounds>
            <type>
                <name>Bar</name>
                <type-bound>Iterator</type-bound>
            </type>
        </bounds>
        <where>
            <type-clause>
                <type></type>
                <type-bound>Clone</type-bound>
            </type-clause>
        </where>
    </trait>
}

fn main() {}
