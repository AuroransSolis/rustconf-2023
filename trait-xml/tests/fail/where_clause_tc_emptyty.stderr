error: error parsing type: empty type tags. caller: `::trait_xml::trait_xml_parse_type_clause`
  --> tests/fail/where_clause_tc_emptyty.rs:1:1
   |
1  | / trait_xml::trait_xml! {
2  | |     <trait>
3  | |         <name>Foo</name>
4  | |         <bounds>
...  |
16 | |     </trait>
17 | | }
   | |_^
   |
   = note: this error originates in the macro `$crate::trait_xml_parse_type_ty` which comes from the expansion of the macro `trait_xml::trait_xml` (in Nightly builds, run with -Z macro-backtrace for more info)
